src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -629,7 +630,11 @@ public bool IsFrozen
         [JsonIgnore]
         public bool IsTransient
         {
-            set { NodeModel.IsTransient = value; }
+            set
+            {
+                NodeModel.IsTransient = value;
+                RaisePropertyChanged("IsTransient");
```

Comment:
You can use nameof(IsTransient) here incase the property got renamed in the future